Client/game_sa/CFileLoaderSA.cpp

```diff
@@ -159,3 +144,87 @@ RpAtomic* CFileLoader_SetRelatedModelInfoCB(RpAtomic* atomic, SRelatedModelInfo*
     }
     return atomic;
 }
+
+bool IsMatrixNeeed(SFileObjectInstance* pInst, CEntitySAInterface* pEntity)
```

Comment:
This function does more than it's name says.
I think it should be called `MaybeNormalize<matrix?>`